docs/dev/new_check_howto.md

```diff
@@ -475,11 +479,11 @@ The `service_checks.json` file contains the following mandatory attributes:
 | Attribute       | Description                                                                                                              |
 | ----            | ----                                                                                                                     |
 | `agent_version` | Minimum Agent version supported.                                                                                         |
-| `integration`   | Integration name.                                                                                                        |
+| `integration`   | The name of the integration that emits this service check. Must be the `display_name` from the manifest.json.                                                                                                      |
```

Comment:
Does that mean we should use the display_name directly, or it's escaped version like in `metadata.csv`?

Reply:
Display_name directly for now.

Reply:
added `non normalized`